### DISCUSSION TITLE
New API Dashboard Get Session

### DISCUSSION THREAD

**Discussion by justinTM** [grafana/grafana#40231] [Dashboards, Panels & Visualizations]
Upon maintenance of dashboards, I need to know who (if anyone) is currently using a dashboard before making it unavailable during redeployment. I want to notify any users of impending outage.

It appears (please correct me if I'm wrong) there is no API endpoint to handle user sessions; it would be a nice feature. 

"Another session is editing this dashboard"
![image](https://user-images.githubusercontent.com/9123665/136588411-42d8dbe2-bca8-465b-ad29-598fe43fe1b4.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
